rust/generate_message/src/helpers.rs

```diff
@@ -686,6 +688,38 @@ pub fn generate_key_info_export_to_qr<P: AsRef<Path>>(
     generate_qr_code(&export_addrs_encoded, chunk_size, fps, output_name)
 }
 
+/// Generate Bulk Transaction Signing payload for testing
+pub fn generate_bulk_transaction_qr<P: AsRef<Path>>(
+    dst_file: P,
+    tx_count: usize,
+    chunk_size: u16,
+    from: String,
+    output_format: Goal,
+) -> Result<()> {
+    let encoded_transactions = (0..tx_count).map(|_| {
+
+let line = "0102".to_string() + 
+    &from +
+"a80403004adb5312dbd3a1bc28610deb1fb631110bbcccb6e7fb97e18501509d5565ea760b00a014e3322615010000682400000e000000e143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e2faa9938ec10c2627d0fd5d20214aba3bf281e94a6a8f50d19b4e0ce3b253d65e143f23803ac50e8f6f8e62695d1ce9e4e1d68aa36c1cd2cfd15340213f3423e";
+        hex::decode(line).unwrap()
+    })
+    .collect();
+    let v1_bulk = TransactionBulkV1 {
+        encoded_transactions,
+    };
+    let bulk = TransactionBulk::V1(v1_bulk);
+
+    let payload = [&[0x53, 0xff, 0x04], bulk.encode().as_slice()].concat();
+
+    match output_format {
+        Goal::Qr => generate_qr_code(&payload, chunk_size, 8, dst_file)?,
```

Comment:
It's not obvious what `8` means. Maybe make it a constant?

Reply:
I believe that the changes to `generate_message` can be rolled back before merging this anyway but yes